src/audio/drc/drc.c

```diff
@@ -235,6 +266,16 @@ static void drc_set_alignment(struct audio_stream *source,
 	audio_stream_init_alignment_constants(1, 1, sink);
 }
 
+static bool drc_has_config(struct drc_comp_data *cd)
```

Comment:
"static inline"

Reply:
on ACE and cAVS the compiler is now allowed to inline automatically. If this is an issue on other platforms, I'd rather enable `CONFIG_COMPILER_INLINE_FUNCTION_OPTION` on them too, than adding `inline` to individual functions.